app/src/main/kotlin/org/gradle/tools/trace/app/TraceToChromeTraceConverter.kt

```diff
@@ -79,36 +77,33 @@ class TraceToChromeTraceConverter : BuildOperationVisitor {
 
         events.add(TracePacket.newBuilder()
             .setTimestampClockId(64)
-            .setTimestamp(record.startTime - startTime.get())
+            .setTimestamp(start.startTime - startTime.get())
             .setTrustedPacketSequenceId(1)
             .setTrackEvent(TrackEvent.newBuilder()
                 .setTrackUuid(uuid)
-                .setName(record.displayName)
-                .addCategories(record.detailsClassName ?: "")
-                .addCategories(record.resultClassName ?: "")
```

Comment:
Why no longer store result info?

I imagine this would be useful to view in the visualizer

Reply:
The result info (the debug info) is still stored, though when the finish event comes in. When I add a category in the finish event, then Perfetto doesn't detect it as the end of the started event, so I can't really do that. I think it is also fine, since the type of the result class is determined by the details class anyway.